docs/pages/faq.mdx

```diff
@@ -94,6 +94,27 @@ Please refer to the [Ports](admin-guide.mdx#ports) section of the Admin Manual.
 
 Gravitational offers this feature for the [Enterprise versions of Teleport](enterprise/introduction.mdx).
 
+### What features ship with Teleport?
```

Comment:
Let's hold back on this PR for a couple of reasons:

* Let's try to be consistent in our description of Teleport. Please take a look at the messaging guide here:

https://docs.google.com/document/d/1UX4GNKi2kr6xtfPk27P_kWYME39LAWMg8Z2tOF8mzA0/edit#

It lists benefits, language and terminology

* The feature overview is a great idea, but I think it's better placed in architecture.
* The list of features seems like a duplicate of the introduction's pages.

Reply:
I would also suggest starting with just one PR to review, and then creating backports after it's merge. You will reduce amount of changes/tweaks based on PR comments and it will be more clear for folks what PR to review first.

Reply:
FYI - I will temporarily close this to reduce noise until we're more settled on the best approach here. Thanks!